releasenotes/notes/fix-aws-lambda-circular-dep-when-patching-in-handler-320de20015a8d5cd.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    aws_lambda: Fix AttributeError raised when `ddtrace.patch_all()`, or `ddtrace.patch(aws_lambda=True)`, is set on user handler.
```

can we add a newline character here?
```suggestion
    aws_lambda: Fix AttributeError raised when `ddtrace.patch_all()`, or `ddtrace.patch(aws_lambda=True)`, is set on user handler.
    
```